invenio_rdm_records/services/config.py

```diff
@@ -144,10 +146,18 @@ class RDMRecordServiceConfig(RecordServiceConfig):
                 "system_managed": False,
             },
         },
+        "oai": {
+            "invenio": {
+                "provider": OAIInvenioPIDProvider,
+                "required": True,
+                "system_managed": True,
+            }
+        }
     }
 
     pids_providers_clients = {
-        "datacite": DOIDataCiteClient
+        "datacite": DOIDataCiteClient,
+        "invenio": OAIInvenioClient,
```

Comment:
the name of the pid provider and pid provider client has to be the same and can have a maximum of 8 characters on the database level. this would restrict it to `internal` only. is this enough or could it be misleading?